platform/core/src/classes/CommandsManager.ts

```diff
@@ -161,6 +162,49 @@ export class CommandsManager {
       return commandFn(commandParams);
     }
   }
+
+  /**
+   * Run one or more commands with specified extra options.
+   *
+   * @param toRun - A specification of one or more commands
+   * @param extraOptions - to include in the commands run beyond
+   *   the commandOptions specified in the base.
+   */
+  public run(
+    toRun: Command | Commands | Command[] | undefined,
+    extraOptions?: Record<string, unknown>
```

Comment:
it should just be options, extraOptions is just weird, and I prefer `commandInput` instead of `toRun`

Reply:
No, it isn't options, it is extra options to add in, which is different from options.  But, whatever, will rename as options